convex/experimentMetrics.ts

```
@@ -0,0 +1,582 @@
+    if (!experiment) {
+      throw new Error('Experiment not found')
+    }
+
+    // Determine time window boundaries
+    const now = Date.now()
+    let windowStart: number
+    let windowEnd: number
+
+    switch (args.timeWindow) {
+      case 'daily':
+        const startOfToday = new Date()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)